tests/test_visitors/test_tokenize/test_primitives/test_numbers/test_underscored_numbers.py

```diff
@@ -11,11 +11,19 @@
 @pytest.mark.parametrize(
     'primitive',
     [
-        '333_555',
+        '1_00',
+        '10_00',
+        '100_00',
+        '-1_0',
+        '-10_0',
+        '-100_0',
+        '1000_000',
+        '-1000_000',
+        '1.0_0',
```

Comment:
Please increase coverage of invalid `float` numbers. Add: `0_0.00_1` and `0.000_1`